framework/components/ecl-carousels/test/spec/carousel.js

```diff
@@ -0,0 +1,58 @@
+const componentUrl = 'ecl-carousels.html';
```

Comment:
There seems to be display issues with the screenshots (most of them don't display the pictures).